test/torchaudio_unittest/functional/functional_cpu_test.py

```diff
@@ -1,13 +1,24 @@
+import itertools
 import math
 import warnings
 
 import torch
-import torchaudio
-import torchaudio.functional as F
 from parameterized import parameterized
-import itertools
-
 from torchaudio_unittest import common_utils
+from torchaudio_unittest.common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    save_wav,
+    skipIfNoExec,
+)
+from torchaudio_unittest.sox_io_backend.common import name_func
+
+import torchaudio
+import torchaudio.functional as F
+from torchaudio._internal import (
+    module_utils as _mod_utils,
+)
+from torchaudio.backend import sox_io_backend
```

Comment:
Is it your IDE that is sorting the import statements?
My interpretation of PEP8 here is that `torchaudio_unittest` is the local library so it should be the third group and `torch`, `torchaudio` and other libraries used for testing are the second group.
So I prefer `torchaudio` comes before `torchaudio_unittest`. But if this is something your IDE does automatically, I won't argue.

https://www.python.org/dev/peps/pep-0008/#imports
> Imports should be grouped in the following order:
>
> 1. Standard library imports.
> 2. Related third party imports.
> 3. Local application/library specific imports.
You should put a blank line between each group of imports.